docs/src/corpora/indexedcorpus.rst

```diff
@@ -8,9 +8,3 @@
     :undoc-members:
     :show-inheritance:
 
-
-.. autoclass:: IndexedCorpus
```

Comment:
Merge current `develop` to your PR please (this part will go away, because this is already merged)

Reply:
I merged the changes, and commited back, but I can't get this message to go away.